docs/src/api.md

```diff
@@ -36,6 +36,7 @@ sample!
 ```@docs
 nearby_ids
 nearby_positions
+euclidean
```

Comment:
The reason I suggested `edistance` instead of `euclidean` is because the latter conflicts with Distances.jl. However, given that we use Tuples instead of SVectors people can't use distances with it anyways. I think its okay.

Reply:
Oh, I thought you said euclidean. Must have read that in the email, not here. Easily updated.